tests/unittests/test_bank_account_validation.py

```diff
@@ -42,3 +42,12 @@ def test_bank_account_validation(test_case):
                 validate_bank_account(bank_number, bank_branch_number, account_number)
         else:
             assert validate_bank_account(bank_number, bank_branch_number, account_number) == expected_result
+
+
+def test_one_zerro():
```

Comment:
@Anton6896 this test is redundant since we run the test-cases in /tests/unittests/data/test_bank_validators_data.json.
so you just need to expand there the test case for one-zero :)